proposals/vector_to_points.json

```diff
@@ -0,0 +1,68 @@
+{
+    "id": "vector_to_points",
+    "summary": "Sample random points from geometries",
+    "description": "For each geometry samples a number of random points based on one of the sampling method.",
+    "categories": [
+        "cubes",
+        "vector"
+    ],
+    "experimental": true,
+    "parameters": [
+        {
+            "name": "data",
+            "description": "Input geometries for sample extraction.\n\nTo maximize interoperability, a nested `GeometryCollection` should be avoided. Furthermore, a `GeometryCollection` composed of a single type of geometries should be avoided in favour of the corresponding multi-part type (e.g. `MultiPolygon`).",
+            "schema": [
+                {
+                    "type": "object",
+                    "subtype": "geojson"
+                },
+                {
+                    "type": "object",
+                    "subtype": "vector-cube"
+                }
+            ]
+        },
+        {
+            "name": "count",
+            "description": "The number of points to compute. For each point given the maximum number of computed points is always 1.",
+            "optional": true,
+            "default": 1,
+            "schema": {
+                "type": "integer",
+                "minimum": 1
+            }
+        },
+        {
+            "name": "method",
+            "description": "The method to distribute the samples:\n\n* `random` (default): Random sampling\n* `fibonacci`: Fibonacci spiral sampling",
```

Comment:
I'm not an expert on this, but when a user want's Fibonacci spiral sampling, they might care about CRS of the calculation of the Fibonacci spiral?

Also on the backend side: I think it is far from easy to do Fibonacci sampling in a non-trivial geometry so that you get exactly the requested number of samples.

Reply:
Me neither, so I would appreciate thoughts from others, e.g. @mattia6690 

We could also just leave out the method for now.

Reply:
Okay, I can remove Fibonacci. Should we include some kind of a "regular" option instead?

Reply:
> important to have a regular (e.g. one point every x meters) ... sampling 

@mattia6690  Does that mean that the spatial density of sample points  is more important than their  number? If so, the `count` (or `size`) argument might be annoying in practice  for Fibonacci or other regular sampling methods .

Reply:
> @mattia6690 Does that mean that the spatial density of sample points is more important than their number? If so, the `count` (or `size`) argument might be practically unusable for Fibonacci or other regular sampling methods .

It depends... you can have a regular grid also based on a size argument. The smaller the size the bigger the gaps between the points. The same is true for the spiral fibonacci sampling. You have the same sampling structure but with a less dense cloud of points with a smaller `count` (or `size`) argument. Even though I would really remove Fibonacci for the moment...